mmdet/models/detectors/semi_base.py

```diff
@@ -69,6 +69,21 @@ def reweight_loss(losses: dict, weight: float) -> dict:
                     losses[name] = loss * weight
         return losses
 
+    @staticmethod
```

Comment:
we can implement a util function in utils and use it in places outside semi-detector

Reply:
There exits a similar function in https://github.com/open-mmlab/mmdetection/blob/c9b47820d637ab341ca3ca869116fc3d91030c74/mmdet/models/utils/misc.py#L292, this function is just convenient called by subclasses of semi-detector.

Reply:
rename_loss is more common, it can be used in many places like DETR.